gebru/roster.json

```diff
@@ -67,8 +67,8 @@
       "github": "iAmash412"
     },
     {
-      "name": "Alfred Kelly",
-      "github": "Scralfie"
+      "name": "Jianli Lin",
```

Comment:
I honestly don't know why it did not put the curly brackets, my only guess it is because it did not sync when Alfie pulled his request, I have (I think at least) send the request again with the correct format, and I hope it worked

Reply:
Hi Jianli, this didn't need to take you the whole day. Do ask for help next time :)

Reply:
I definitely will!